prompts/proposalPolice.ts

```diff
@@ -0,0 +1,23 @@
+import CONST from '@github/libs/CONST';
+
+export default class ProposalPoliceTemplates {
```

Comment:
NAB but I'm not sure it's helpful to use a class in this context. A plain-old object, exported `as const` seems like it would do the trick.